memory.cpp

```diff
@@ -141,14 +143,16 @@
     #else
         const int frames_to_skip = 7;
     #endif
-        // int trace_size = backtrace(trace, stack_depth);
-        // for (int i = frames_to_skip; i < trace_size; ++i)
-        // {
-        //     Dl_info dl_info = {};
-        //     dladdr(trace[i], &dl_info);
-        //     if (dl_info.dli_fname && strstr(dl_info.dli_fname, "/IntelSEAPI."))
-        //         return true;
-        // }
+        #ifndef USE_MUSL_LIBC
+            int trace_size = backtrace(trace, stack_depth);
```

Comment:
Why was this commented out before and now it is not. If USE_MUSL_LIBC is not defined should this code really be executed?